src/print/handlers.ts

```diff
@@ -1036,7 +1036,7 @@ const handlers: Record<string, Handler> = {
 
 		const key = handle(node.key, state);
 
-		if (node.method || (node.value.type === 'FunctionExpression' && !node.value.id)) {
+		if (node.value.type === 'FunctionExpression' && !node.value.id) {
```

Comment:
This change is just unrelated nearby tidying. I'm pretty sure if `node.method` is true, then its `value` will have to be a `FunctionExpression` with no `id`. The reason for the `method` boolean seems to be so that we can tell the difference in the AST between `{ foo() {} }` and `{ foo: function() { } }`, which we don't care about.